libs/i18n/src/lib/languages/albanian.ts

```diff
@@ -137,6 +137,9 @@ export const FD_LANGUAGE_ALBANIAN: FdLanguage = {
         deleteButtonLabel: 'E fshishme',
         ariaRoleDescription: 'shenjë'
     },
+    coreTokenizer: {
+        moreLabel: '{{count}} more'
```

Comment:
just an idea - maybe we can have some mark in not translated yet strings? like `[TODO]` so then when someone will be translating it it'll be easier to find these strings.